Source/VMware.vSphereDSC/Tests/Unit/TestHelpers/VMware.VimAutomation.Core/VMware.VimAutomation.Core.psm1

```diff
@@ -463,6 +463,33 @@ Add-Type -TypeDefinition @"
             return (this.Name + "_" + this.User).GetHashCode();
         }
     }
+
+    public class SyslogConfig
```

Comment:
Implement System.IEquatable<T> Interface here.

Reply:
Ok, I understand.
But wouldn't it than be better to implement that interface always?
Or is there a drawback?

Reply:
Well I think in the case of SATP Claim Rule it creates little overhead, because you have to write three methods to implement the interface when in the unit tests you do not need to compare the SATP instances.

Reply:
Left it out as I don't use the Type as a parameter